src/Cluster.jl

```diff
@@ -370,4 +377,225 @@ function fit!(model::BKMeans, X)
     end
 end
 
+
+
+mutable struct DC
+    k::Int
+    mode
+    max_try::Int
+    tol::Float64
+    centroids::Array{Float64,2}
+    labels::Array{Int,1}
+end
+
+"""
+    compute_objective_function(data, centroids,k) -> Array
+
+Computes the distance from each data point to each centroid.
+
+# Arguments
+- `data`: The input data matrix where each row is a data point.
+- `centroids`: The current centroids.
+- `k` the exponent of norm 
+# Returns
+A distance matrix `D` of size (number of data points, number of centroids), where `D[i, j]` is the distance from data point `i` to centroid `j`.
+
+# Examples
+```julia-repl
+X = [
+    1.0 1.0;
+    1.5 2.0;
+    3.0 4.0
+]
+centroids = [
+    1.0 1.0;
+    3.0 4.0
+]
+D = compute_distance(X, centroids)
+```
+"""
+function compute_objective_function(X, centroids,k)
+    x = size(X)
+    y = size(centroids)
+    D = zeros(x[1], y[1])
+    delta = 0.0001
```

Comment:
this should probably an optional function parameter. (Don't really know the algorithm 😉 )